app/schema/portal/cdo/microchip-search.js

```diff
@@ -0,0 +1,32 @@
+const Joi = require('joi')
+
+const validNewMicrochip = /^[0-9\s]+$/
```

Comment:
Just to check, this would allow ``232323 21321323`` to pass as permits spaces.  I'm assuming that's correct.  Do we remove the spaces for the api call?

Reply:
if we export the validateMicrochip fn we could just unit test on this, maybe?